### DISCUSSION TITLE
Mod configuration

### DISCUSSION THREAD

**Discussion by CyberSteve777** [GrassyDev/PvZCubed#13] [Q&A] [answered]
Hello! Is it possible to configure mod to hide "gay" textures, for ex.? I just want not to see them

**Comment by GrassyDev** [comment]
You can remove them using white dye.
I may add a toggle when I add configs to this mod but this is not a priority.

**Comment by GrassyDev** [comment, marked as answer]
I thought about it and I will not add a toggle on the configs to 'hide "gay" textures, for ex.'
The next update will alleviate the problem of your garden being way too colorful by making the skins rarer. But they will be there and if it breaks the PvZ immersion, using white dye on them will still be an option.
